webpack/assets/javascripts/react_app/components/LoginPage/LoginPageFooter.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { Button } from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+import { translate as __ } from '../../common/I18n';
+
+const LoginPageFooter = ({ caption }) => {
+  const [showLess, setShowLess] = useState(true);
+  if (!caption) return null;
+  let loginCaption = <p>{caption}</p>;
+
+  if (caption.length > 500) {
+    loginCaption = (
+      <>
+        <p>{showLess ? `${caption.substring(0, 500)}...` : caption}</p>
```

Comment:
^^ here for example

Reply:
:+1: for having constants. Back in university my programming professor would fail us on this.

Reply:
Sure, I will add it :)
Do you have a good name for it?
how about `LONG_TEXT_BREAK_CHAR`?

Reply:
Yeah, the naming is the hardest part :D

What about `MAX_LONG_TEXT`?

Reply:
feels a bit too generic, @Mariaaga @amirfefer @ekohl any ideas? :D

Reply:
Honestly I'm not even convinced the show more/show less is needed at all. There appears to be plenty of space so why do we need to do it? Users can scroll if needed. I'd vote for removal and always show the full text.

Reply:
Good question @ekohl, I will speak with @marisvirik and @rohoover about it,
let's see what they think...

some of the text can be missed if the user didn't scroll, and a button might be more clear... though I am not against your idea :)

Reply:
@laviro @ekohl If there are no law requirements to show the entire text at once, I would stick with Read more option even though we have enough space to show it. 
1. Keeping the entire text visible would make the Login page look crowded and busy. 
2. As a user I would be confused if it's important for me to read the text in order not to miss any essential information for my next actions. Fistly, as a user, I would find the text non-essential because of its tiny font size and location in the footer. On the other hand, it's too long to just ignore it. So I believe it could create some kind of psychological distress on the page where we want users just to focus on logging in.

In conclusion, my two cents are that we don't want to overwhelm the users on the very first page with a long text area with minimum font size, which is not essential for their next actions. So keep Read more truncation.

But we can still work on the content of the actual text with the Content team and create a text variant when the Read more want be necessary and the text won't be too long.

Reply:
I don't understand this. By default there is no text at all. There is a setting that is the login text but the admin controls this value. That means that any text that is there should be deemed important to an admin, right? The admin may even be annoyed by the Read More button.

I remain unconvinced that there's positive value in a Read More button for users and for us as maintainers there's a negative value.

Reply:
the `Read more` will be visible only if the caption length is more than 500 chars long,
I am not sure how long users usually write those texts, maybe we should make some limit to this setting,
so we will know for sure that the users will write a max of 1000 chars for example, so we can adjust the page,
with 1000 chars I don't think we need the Read More/Less.

btw Twitter uses 280 char limit :P

Reply:
@ekohl I apologize, I got confused about the main goal of the text (its creation, etc.), I thought it's just some kind of a prefabricated legal disclaimer. Thank you @laviro for an explanation. My proposal is to keep it this way until we get some feedback from customers so our decisions are based on data, not our assumptions.

Reply:
I think we should drop `Read More` until there's a need for it. We never had it. That would keep this PR scoped to just "move" and not "move + add max char limit". Combining would also not be clear for users who read the changelog and thus be a surprise for upgrading users.

Reply:
Sounds good, if we'll need it, we can add it in another PR..